html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue

```diff
@@ -119,6 +225,174 @@ export default {
           label: this.$i18n.t('Owner'),
           sortable: true,
           visible: true
+        },
+        {
+          key: 'last_ip',
+          label: this.$i18n.t('IP Address'),
+          sortable: true,
+          visible: true
+        },
+        {
+          key: 'tenant_name',
+          label: this.$i18n.t('Tenant'),
+          sortable: true,
+          visible: true
+        },
+        {
+          key: 'device_class',
+          label: this.$i18n.t('Device Class'),
+          sortable: true,
+          visible: true
+        },
+        {
+          key: 'device_score',
+          label: this.$i18n.t('Device Score'),
+          sortable: true,
+          visible: false
+        },
+        {
+          key: 'device_type',
+          label: this.$i18n.t('Device Type'),
+          sortable: true,
+          visible: true
+        },
+        {
+          key: 'device_version',
+          label: this.$i18n.t('Device Version'),
+          sortable: true,
+          visible: false
+        },
+        {
+          key: 'dhcp6_enterprise',
+          label: this.$i18n.t('DHCPv6 Enterprise'),
+          sortable: true,
+          visible: false
+        },
+        {
+          key: 'dhcp6_fingerprint',
+          label: this.$i18n.t('DHCPv6 Fingerprint'),
+          sortable: true,
+          visible: false
+        },
+        {
+          key: 'dhcp_fingerprint',
+          label: this.$i18n.t('DHCP Fingerprint'),
+          sortable: true,
+          visible: false
+        },
+        {
+          key: 'dhcp_vendor',
+          label: this.$i18n.t('DHCP Vendor'),
+          sortable: true,
+          visible: false
+        },
+        {
+          key: 'category_id',
+          label: this.$i18n.t('Role'),
+          sortable: true,
+          visible: true
+        },
+        {
+          key: 'switch_id',
```

Comment:
I think none of the `switch` fields can currently be retrieved through the `/nodes/search` API call.